simulation/optimization/genetic.py

```diff
@@ -424,7 +424,7 @@ def fitness(self, ga_instance, solution, solution_idx) -> float:
             else self.model.get_distance_before_exhaustion()
 
         self.did_finish_race = True if distance_travelled_real == distance_travelled else False
-
+        fitness = (691200 / time_taken) * (distance_travelled_real / 2466)
```

Comment:
Max time and max distance? 
May be good to make these parameters, since we may start on day 2 or something. I imagine fitness may be changing in the future anyways.